### DISCUSSION TITLE
How to show the most recent notification on top of the history?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [folke/snacks.nvim#147] [Q&A] [answered]
I would like to see notifications in reversed order (latest at the top). Is it possible?

**Comment by folke** [comment]
Check the docs for the sort option.
By default it shows notifs top-down by severity and then time

**Reply by wojciech-kulik** [reply]
Yes, I checked it before posting. `top_down` option changes the placement of notification pop-ups, not the order in the history. If I switch `top_down` to `false`, notifications appear in the right-bottom corner instead of the top-right.

**Reply by wojciech-kulik** [reply]
Oh, sorry I thought you are refering to `top_down` option. Regarding, `sort` I don't see any information on how to reverse the order?

**Reply by folke** [reply]
you can't put the most recent on top, since that would mean your visible notifications would constantly change places.
Which is something nobody wants I think.

**Reply by wojciech-kulik** [reply]
I think you are referring to pop-up and I'm referring to notification history :). We don't have the ability to reverse the order and we don't have the ability to clear the history, so if you have lots of notifications, every time you open notification history you have to go to the bottom to see the latest one, quite tedious.

**Reply by wojciech-kulik** [reply]
Unless you are talking about live updates in notification history. In that case, it would also be useful to be able to disable live updates. 
Update: I see that the history is not refreshed automatically, so it shouldn't be a problem.

**Reply by folke** [reply, marked as answer]
Just added a `reverse` option!

**Reply by wojciech-kulik** [reply]
Works perfectly! Thank you very much for the great plugin, a great improvement to my Neovim workflow!
